pom.xml

```diff
@@ -2,50 +2,47 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.rdfhdt</groupId>
   <artifactId>hdt-mr</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>2.0</version>

```

Comment:
I compiled it with dependency of hdt-java 2.0, so I called it 2.0….I think we can call it 2.0 alpha? Anyway, it compiled well, but I have not yet tested it…

From: Arto Bendiken [mailto:notifications@github.com]
Sent: Wednesday, June 22, 2016 2:30 PM
To: rdfhdt/hdt-mr <hdt-mr@noreply.github.com>
Cc: Fu, Gang (NIH/NLM/NCBI) [E] <gang.fu@nih.gov>; Author <author@noreply.github.com>
Subject: Re: [rdfhdt/hdt-mr] add pom.xml to compile codes dependency includes hdt-java 2.0 (#2)


In pom.xml<https://github.com/rdfhdt/hdt-mr/pull/2#discussion_r68108593>:

> @@ -2,50 +2,47 @@

>    <modelVersion>4.0.0</modelVersion>

>    <groupId>org.rdfhdt</groupId>

>    <artifactId>hdt-mr</artifactId>

> -  <version>1.0.0-SNAPSHOT</version>

> +  <version>2.0</version>

Why the bump to 2.0? The software remains "alpha quality", according to its authors.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/rdfhdt/hdt-mr/pull/2/files/08de9a4abbd836f593502fb68f73b1d4ea8f147f..88f81392bcfff195666f69f4da2740b4432340e8#r68108593>, or mute the thread<https://github.com/notifications/unsubscribe/AEAifKfOTFUjyJ4xKU9V6kzohKQ40Pjjks5qOX8NgaJpZM4I8EF9>.